kube-derive/src/custom_resource.rs

```diff
@@ -34,7 +35,12 @@ struct KubeAttrs {
     printcolums: Vec<String>,
     #[darling(multiple)]
     selectable: Vec<String>,
-    scale: Option<String>,
+
+    /// Customize the scale subresource, see [Kubernetes docs][1].
+    ///
+    /// [1]: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource
+    scale: Option<Scale>,
```

Comment:
if the dependency is just for [this struct](https://docs.rs/k8s-openapi/latest/k8s_openapi/apiextensions_apiserver/pkg/apis/apiextensions/v1/struct.CustomResourceSubresourceScale.html), then maybe we should inline that for the interface instead. It would also avoid having to do the newtyping you do below.